### PR TITLE
added new component Todo List Item, added handle to Add Todo Form, an…

### PR DESCRIPTION
**Pull request by Mariela-t**
…d stored New Todo in react state

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,14 +1,22 @@
 import React from "react";
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+  function handleAddTodo(event) {
+    event.preventDefault();
+    let todoTitle = event.target[0].value;
```
Comment (kwilson05) [inline src/AddTodoForm.js:6]: Nice nice!! Maybe use a `const` here. But still a nice job, Mariela

File: src/TodoList.js [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+
   return (
     <div>
-      <ul>{list}</ul>
+      <ul>
+        {todoList.map((item) => (
+          <TodoListItem key={item.id} todoList={todoList} item={item} />
+        ))}
+      </ul>
     </div>
```
Comment (kwilson05) [inline src/TodoList.js:26]: Good job here too! 🎖️ 

Watch out here, you don't need to pass the whole list to each item. What do you think you need to change here?

File: src/App.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
-import React from "react";
+import React, { useState } from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
+  const [newTodo, setNewTodo] = useState();
```
Comment (kwilson05) [inline src/App.js:6]: set that state 💯

**Review by kwilson05 — APPROVED** [review]
Good job Mariela!! 🤽 I'm approve this but I still want you to check out the changes I suggested 😄 

Keep up the good work!!!

File: src/TodoList.js (hunk 2 above)
Comment (Mariela-t) [reply src/TodoList.js:26]: delete todoList={todoList} ?

Comment (kwilson05) [reply src/TodoList.js:26]: Yup yup!! 💯

**Pull request merged**